### ISSUE TITLE
Add Support for Parameter Objects with Content Property

### ISSUE THREAD

**Issue by peombwa** [Azure/autorest#4480]

**Comment by timotheeguerin** [comment]
Duplicate of #4303, is this a demand from services? I did brings this up in autorest scrum a while back and there wasn't much interest in supporting it in the generators without a real demand.

**Comment by peombwa** [comment]
@timotheeguerin, yes, this is a demand from a service. MS Graph API supports `GET /deviceManagement/microsoft.graph.getRoleScopeTagsByIds(ids=["1","2","3"])`, which can only be described with a parameter object of `Content.application/json`. See the discussion in https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/902#issuecomment-1069133352.
